### DISCUSSION TITLE
CSS for colors of multiple tabs selection

### DISCUSSION THREAD

**Discussion by Kratorator** [piroor/treestyletab#3245] [Q&A] [answered]
I can't find the CSS for changing the background color and opacity of selected multiple tabs. With light backgrounds, it's a light yellow, but with dark tab backgrounds (up to middle grey), set by the Firefox theme and/or the Dark Reader addon, it's a barely visible, almost transparent turquoise. 
Changing the text color would be nice too.

**Comment by irvinm** [comment, marked as answer]
> I can't find the CSS for changing the background color and opacity of selected multiple tabs. With light backgrounds, it's a light yellow, but with dark tab backgrounds (up to middle grey), set by the Firefox theme and/or the Dark Reader addon, it's a barely visible, almost transparent turquoise. Changing the text color would be nice too.

For the tab color and opacity, modify the values to what you want.

```
:root {
  --multiselected-color: var(--tab-surface-active-gradient);
  --multiselected-color-opacity: 0.35;
}
```

For the font, modify what you need from here:

```
.mutiple-highlighted > tab-item.highlighted tab-label .label-content {
  font-weight: normal;
  line-height: 1.4;
  color: red;
  text-align: start;
  text-shadow: none;
  margin-bottom: 1px;
  margin-left: 0.25em;
  padding: 0 0 0.25em 0;
}
```

I haven't really tested this, but thought it was a start ...

**Reply by Kratorator** [reply]
Thanks a lot, it works great!
I only changed the background to the same color as my active tab color, at 0.33 opacity.
